hummingbot/client/ui/interface_utils.py

```diff
@@ -111,5 +111,10 @@ def format_df_for_printout(
         )
         df.columns = [c if len(c) < max_col_width else f"{c[:max_col_width - 3]}..." for c in df.columns]
     table_format = table_format or global_config_map.get("tables_format").value
-    formatted_df = tabulate(df, tablefmt=table_format, showindex=index, headers="keys")
+
+    tabulate.PRESERVE_WHITESPACE = True
+    try:
+        formatted_df = tabulate.tabulate(df, tablefmt=table_format, showindex=index, headers="keys")
+    finally:
+        tabulate.PRESERVE_WHITESPACE = False
```

Comment:
I think it would be better to store the current value of `tabulate.PRESERVE_WHITESPACE` to then restore it in the `finally` clause, instead of assuming it was `False` originally

Reply:
Addressed